server/src/main/java/org/jboss/as/server/operations/HttpManagementAddHandler.java

```diff
@@ -185,6 +183,10 @@ protected void installServices(OperationContext context, HttpInterfaceCommonPoli
             RemotingHttpUpgradeService.installServices(serviceTarget, ManagementRemotingServices.HTTP_CONNECTOR, httpConnectorName,
                     ManagementRemotingServices.MANAGEMENT_ENDPOINT, commonPolicy.getConnectorOptions());
         }
+        if (socketBindingServiceName != null) {
+            return Arrays.asList(socketBindingServiceName, UndertowHttpManagementService.SERVICE_NAME);
```

Comment:
UndertowHttpManagementService depends on socketBindingServiceName so only returning UndertowHttpManagementService is sufficient.